admin.py

```diff
@@ -381,7 +381,10 @@ def post(self, path, feature_id=None):
       common.handle_401(self.request, self.response, Exception)
       return
 
+    spec_repo_url = self.__FullQualifyLink('spec_repo_url')
     spec_link = self.__FullQualifyLink('spec_link')
+    security_review_url = self.__FullQualifyLink('security_review_url')
```

Comment:
I think we will have the internal sec review in the launch bug, and we don't need one here (there's no current W3C group doing horizontal review for security).  The TAG review request includes a security and privacy self-review, which should be good enough.  We can remove this.